### PR TITLE
[14.0][ADD] sale_coupon_delivery_auto_refresh

### PR DESCRIPTION
**Pull request by ivs-cetmix**
A bridge module that fixes the following issue:
When both modules are installed and there is a promotion with free shipping to apply automatically to SO, only delivery expense line is added to SO and not the free shipping discount line

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
Functional review ok!

**Review by SicurSam — APPROVED** [review]
Functional review ok!

**Review by elvise — APPROVED** [review]
LGTM!

**Comment by elvise** [comment]
@OCA/crm-sales-marketing-maintainers can review it ?

**Comment by francesco-ooops** [comment]
@OCA/crm-sales-marketing-maintainers please?

**Comment by elvise** [comment]
@OCA/crm-sales-marketing-maintainers  is there a good soul who can make a review? :)

**Comment by rousseldenis** [comment]
@OCA/core-maintainers I see this one is not under any PSC umbrella. Is it wanted or... ?

**Comment by rousseldenis** [comment]
@simahawk Could you check the sync between OCA and github as I see this:

image


And I have no access

**Comment by elvise** [comment]
@dreispt thanks !
If you have time check this porting #53 :)

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
```
Comment (simahawk) [inline oca_dependencies.txt:1]: FTR this file is not needed as dependencies are installed via PIP

**Comment by simahawk** [comment]
> @simahawk Could you check the sync between OCA and github as I see this:
> 
> image
> 
> And I have no access

weird, it's linked to sales* team on our odoo but the team is not applied on GH. I'll check.

**Comment by elvise** [comment]
@simahawk can you check this porting #53 ?

File: oca_dependencies.txt (hunk 1 above)
Comment (elvise) [reply oca_dependencies.txt:1]: @isserver1 can you fix this ?

**Pull request merged**